manager/dispatcher/assignments.go

```diff
@@ -291,26 +307,26 @@ func (a *assignmentSet) message() api.AssignmentsMessage {
 
 // secret populates the secret value from raft store. For external secrets, the value is populated
 // from the secret driver.
-func (a *assignmentSet) secret(readTx store.ReadTx, task *api.Task, secretID string) (*api.Secret, error) {
+func (a *assignmentSet) secret(readTx store.ReadTx, task *api.Task, secretID string) (*api.Secret, bool, error) {
```

Comment:
I would suggest updating the function comments to indicate the newly added return type. For extra credit, can I request you to add a comment for the existing return values :)

Reply:
Will this do?
```golang
// secret populates the secret value from raft store. For external secrets, the value is populated
// from the secret driver. The function returns: a secret object; an indication of whether the value
// is to be reused across tasks; and an error if the secret is not found in the store, if the secret
// driver responds with one or if the payload does not pass validation.
```